### PR TITLE
Devx 3305 auctions restrict keys access manager

### PR DESCRIPTION
**Pull request by darryncampbell-pubnub**
- Changed the PubNub Pub/Sub keys (I couldn't find the old ones in our devrel repo, perhaps they were mistakenly stored in a personal account?)
- Protected the PubNub keys with Access Manager
- Removed the login logic with auth0

### CODE REVIEW

**Review by oliverfcarson — APPROVED** [review]
Changes look good and I think it makes sense to disable auth0 login. I remember it gave a lot of past problems and it seems like it is continuing to do so. I think it makes sense like you've said to leave it in the tutorial, but hide in the demo. I would just recommend two things before merging:

1. You should hide the action for "Login to Bid" in bidding.js:
`        actionCompleted({action: 'Login To Bid', windowLocation: orgwindowlocation}); // This is for the interactive demo on PubNub.com. It is not part of the demo of this application. 
`
2. Since the repo is public, I would add in the GitHub readme and in the tutorial (text, not video) that you've disable auth0 for demo purposes.

No need for another review for the changes unless you want to discuss before merging. Thanks for the updates on this!

**Comment by darryncampbell-pubnub** [comment]
Thanks @olivercarson-pubnub , I have updated the Readme to remove references to auth0 and have removed 'Login to bid' as an action.  I'll update the tutorial too.

**Pull request merged**
